configuration/eclipseiotmarket.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: documentation
+title: Eclipse IoT Marketplace
+---
+
+{% include base.html %}
+
+# Eclipse IoT Marketplace Integration
+
+The [Eclipse IoT Marketplace](https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title) hosts third-party add-ons that can be browsed and installed to openHAB through this integration.
+
+## Installation
+
+Install the market integration from Add-ons → Misc → Eclipse IoT Market.
+
+![iot marketplae addon](images/eclipseiotmarket_addons.png)
+
+## Usage
+
+When you now refresh your browser, you will see new entries in the "Bindings" tab of the Add-ons menu. As the new entries come with logos and descriptions, the card layout (instead of the list layout, can be selected in the top right) is best for browsing the entries. To filter the view to show only third-party add-ons from the marketplace, you can simply type "market" in the search bar:
+
+![entries](images/eclipseiotmarket_listings.png)
+
+Simply use the install/uninstall buttons to add or remove add-ons from your installation.
+Please note that after a system upgrade the installed add-ons are lost and you will need to re-install them. This behavior will be changed in future.
+
+As an alternative installation method, you can browse the catalog on the [Eclipse Marketplace website](https://marketplace.eclipse.org/taxonomy/term/4988%2C4396/title) and use the "Drag to Install" feature by dragging the "Install" button from the web page to your add-ons page of the Paper UI.
+
+## Configuration
+
+The marketplace integration can be configured to only provide add-ons of certain types (Note: Currently only bindings are supported!) and of a certain maturity level.
+
+To do so, you can add a configuration file `conf/services/marketplace.cfg` with content like:
+
+```
```

Comment:
Ideally all markdown code blocks should have an explicit language tag. This one here would be "conf"